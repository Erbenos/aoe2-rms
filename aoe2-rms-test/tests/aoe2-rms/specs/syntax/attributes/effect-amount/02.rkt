#lang aoe2-rms
; Houses support 10/15/20/25 population in dark/feudal/castle/imperial age.

<PLAYER-SETUP>
(effect-amount 'SET_ATTRIBUTE 'HOUSE 'ATTR_STORAGE_VALUE 10)
(effect-amount 'SET_ATTRIBUTE 'HOUSE_F 'ATTR_STORAGE_VALUE 15)
(effect-amount 'SET_ATTRIBUTE 'HOUSE_C 'ATTR_STORAGE_VALUE 20)
(effect-amount 'SET_ATTRIBUTE 'HOUSE_I 'ATTR_STORAGE_VALUE 25)
