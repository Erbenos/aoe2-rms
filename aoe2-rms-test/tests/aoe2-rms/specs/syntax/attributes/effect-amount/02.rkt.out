<PLAYER_SETUP>
effect_amount SET_ATTRIBUTE HOUSE ATTR_STORAGE_VALUE 10
effect_amount SET_ATTRIBUTE HOUSE_F ATTR_STORAGE_VALUE 15
effect_amount SET_ATTRIBUTE HOUSE_C ATTR_STORAGE_VALUE 20
effect_amount SET_ATTRIBUTE HOUSE_I ATTR_STORAGE_VALUE 25
